### ISSUE TITLE
read_sheet Index 1 must have length 1, not 2 error

### ISSUE THREAD

**Issue by ogs22** [tidyverse/googlesheets4#21]
I'm seeing an error after purrr has been updated to version 0.3.0

```r
> library(googlesheets4)
> read_sheet(sheets_example("mini-gap"), sheet = 2)
Error: Index 1 must have length 1, not 2
Call `rlang::last_error()` to see a backtrace
> rlang::last_error()
<error>
message: Index 1 must have length 1, not 2
class:   `purrr_error_bad_element_length`
backtrace:
 1. googlesheets4::read_sheet(sheets_example("mini-gap"), sheet = 2)
 8. purrr:::stop_bad_element_length(...)
 9. purrr:::stop_bad_length(...)
Call `rlang::last_trace()` to see the full backtrace
> rlang::last_trace()
    █
 1. ├─googlesheets4::read_sheet(sheets_example("mini-gap"), sheet = 2)
 2. │ └─googlesheets4:::get_cells(...)
 3. │   └─googlesheets4::sheets_get(ssid)
 4. │     └─googlesheets4:::sheets_spreadsheet(resp)
 5. │       └─purrr::map(ours_theirs, ~pluck(x, .x))
 6. │         └─googlesheets4:::.f(.x[[i]], ...)
 7. │           └─purrr::pluck(x, .x)
 8. └─purrr:::stop_bad_element_length(...)
 9.   └─purrr:::stop_bad_length(...)

```

**Comment by juliangilbey** [comment]
This has been fixed in commit e0f9eca939eb0a756003b03106b108edf1a82b0c and this issue can be closed.

**Comment by ericpgreen** [comment]
I'm seeing a similar (same?) error with `googlesheets4_0.0.0.9000` and `purrr_0.3.2`

```
R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS  10.14
```

```
> rlang::last_error()
<error>
message: Index 1 must have length 1, not 2
class:   `purrr_error_bad_element_length`
backtrace:
 1. googlesheets4::read_sheet(...)
 8. purrr:::stop_bad_element_length(...)
 9. purrr:::stop_bad_length(...)

> rlang::last_trace()
    █
 1. ├─googlesheets4::read_sheet(...)
 2. │ └─googlesheets4:::get_cells(...)
 3. │   └─googlesheets4::sheets_get(ssid)
 4. │     └─googlesheets4:::sheets_spreadsheet(resp)
 5. │       └─purrr::map(ours_theirs, ~pluck(x, .x))
 6. │         └─googlesheets4:::.f(.x[[i]], ...)
 7. │           └─purrr::pluck(x, .x)
 8. └─purrr:::stop_bad_element_length(...)
 9.   └─purrr:::stop_bad_length(...)
```
